core/src/main/kotlin/plugins/Lifecycle.kt

```diff
@@ -50,6 +54,14 @@ fun Application.configureLifecycle() {
                 syncRoles(authorizationService)
             }
         }
+
+        thread {
+            MDC.setContextMap(mdcContext)
```

Comment:
Question: Will this be removed again when the job is done?

Reply:
It's stored in a `ThreadLocal` so it will die with the thread.

Reply:
Sorry, what I meant was the whole block to add the job to the maintenance service. When the migration is complete it is not needed any more. If the job is started, it will probably still do some (minor) background activity.
I guess, the underlying problem here is that the minimum maintenance service implementation does not provide a mechanism to add/remove jobs dynamically; therefore, starting of jobs has to be hard-coded.